workshops/scipy2022/README.md

```diff
@@ -28,6 +28,8 @@ Logistics;
 {doc}`../../fundamentals/01_datastructures`
 
 {doc}`../../fundamentals/01.1_io`
+
+{doc}`/home/runner/work/xarray-tutorial/xarray-tutorial/data_cleaning/ice_velocity.ipynb`
```

Comment:
```suggestion
{doc}`../../data_cleaning/ice_velocity`
```

This was confusing when I first tried to get it to work but I think you need to skip the `.ipynb` extension